setup.py

```diff
@@ -1,78 +1,101 @@
 """
-Setup module for the dask_labextension
+dask_labextension setup
 """
+import json
+import os
+
+from jupyter_packaging import (
+    create_cmdclass, install_npm, ensure_targets,
+    combine_commands,
+)
 import setuptools
-from setupbase import create_cmdclass, ensure_python, find_packages
 
-import os.path
+HERE = os.path.abspath(os.path.dirname(__file__))
+
+# The name of the project
+name="dask_labextension"
```

Comment:
Obviously super minor, but I'm surprised this passes black.

Reply:
Because it only checks the `dask_labextension` folder: https://github.com/ian-r-rose/dask-labextension/blob/jlab3/.github/workflows/python.yaml#L41

Reply:
`black`/`flake8` have never actually been checked in CI for this repo. I've set it up here to do that, but left it disabled for now: https://github.com/ian-r-rose/dask-labextension/blob/283c1d1db31eb9d24e07e0c44796d97bc7d29ea2/.github/workflows/python.yaml#L37

I'd like to flip the switch here in a follow-up PR.